.vscode/settings.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Just wanted to make my own linter happy.  If you don't have these plugins, won't hurt or help, but safe to add regardless.